src/components/tableview/TableTable.vue

```diff
@@ -143,22 +154,42 @@ export default {
       initialSort: this.initialSort,
       cellClick: this.cellClick
     });
+
+    this.addFilter()
   },
   methods: {
     cellClick(e, cell) {
       this.selectChildren(cell.getElement())
     },
+    addFilter() {
+      this.filter.push({
+        value: null,
+        type: "=",
+        field: this.table.columns[0].columnName
+      })
+    },
+    removeFilter(index) {
+      this.filter.splice(index, 1)
+    },
+    resetFilter() {
+      this.filter = []
+      this.addFilter()
+      this.clearFilter()
+    },
     triggerFilter() {
-      if (this.filter.type && this.filter.field) {
-        if (this.filter.value) {
-          this.tabulator.setFilter(
-            this.filter.field,
-            this.filter.type,
-            this.filter.value
-          );
-        } else {
-          this.tabulator.clearFilter();
+      const filterArray = JSON.parse(JSON.stringify(this.filter));
```

Comment:
why do you do this?

Reply:
`this.filter` is a observer, as i already told this is my first vue application so maybe there is a better method to get the object from a observer.
I googled that and this was the solution i found :-)

Reply:
so I don't think you need to worry about converting filter to another data type.

I think you can just pass it as-is `this.tabulator.setFilter(this.filter)`

What I would do is make a computed property called something like `checkedFilter` or `resolvedFilter`, and in there do the validity check.

Then we can use that computed field to do a couple of things:
1. in the Filter UI we can use it to disable the 'go' button if filters are not valid
2. We can just pass that through to tabulator in this method without any checking

Reply:
once this one change is made, I'll merge it, but @ghstng and I need to iterate on the UI a little. We have a multi-filter UI in mind.

Reply:
Thats fine for me, i use a computed function now :-)